### PR TITLE
Added subtitles to Quiz 1 and modified Quiz 3

### PR DESCRIPTION
**Pull request by LeoCumpli21**
## Summary

This PR closes LeoCumpli21/PyZombis#20. I added subtitles to the exercises as agreed with my fellow collaborators.
Also, it closes LeoCumpli21/PyZombis#21. I modified quiz 3 to accomplish the accorded conventions, and I added a ``playwright`` test to it as well. 

## Checklist

- [x] Variables, function and comments are translated to Spanish
- [x] Spell check done & typos fixed
- [x] All python code is PEP8 compliant
- [x] Test coverage with Playwright implemented
- [x] Reviewers assigned (all peers & at least 1 mentor)

## Screenshots

https://user-images.githubusercontent.com/80920858/121257389-3e7b6300-c873-11eb-90ed-1591503a90d1.mp4

### CODE REVIEW

**Comment by NicolasSandoval** [comment]
Good advance, but this is Quiz-1, right?

**Review by angelasofiaremolinagutierrez — COMMENTED** [review]
Everything looks ok to me. Just change the title of the PR to Quiz 1 instead of Quiz 9 as @NicolasSandoval noticed ;)

**Comment by LeoCumpli21** [comment]
> Good advance, but this is Quiz-1, right?

Indeed, I'll change it.

**Comment by LeoCumpli21** [comment]
> Everything looks ok to me. Just change the title of the PR to Quiz 1 instead of Quiz 9 as @NicolasSandoval noticed ;)

Done! :)

**Review by y-martinez — APPROVED** [review]
LGTM! 🚀

**Review by y-martinez — CHANGES_REQUESTED** [review]
I was checking all exercises and I figure out that the sixth has a problem, I didn't check them in my previous review 😅

image

**Review by y-martinez — APPROVED** [review]

**Review by angelasofiaremolinagutierrez — APPROVED** [review]

**Review by reingart — APPROVED** [review]
Muy buen trabajo! y tambien los reviews!

Sugiero agregar los dos checks nuevos a https://github.com/PyAr/PyZombis/blob/master/.github/pull_request_template.md
- [x] lower_case_with_underscores notation for function/methods & variables names (not camelCase, except classes)
- [ ] Use python code for test locators, instead of exercise description or html/css

File: tests/test_quiz1.py [hunk 1]
```diff
@@ -1,17 +1,21 @@
 def test_quiz1_1(page):
-    page.goto("/quiz/Quiz1.html")
+    page.goto("quiz/Quiz1.html")
 
-    page.click("pre[role=\"presentation\"]:has-text(\"​\")")
+    page.click("text=def suma(n,m):")
```
Comment (eamanu) [inline tests/test_quiz1.py:4]:
```suggestion
    page.click("text=def suma(n, m):")

```

File: tests/test_quiz1.py [hunk 2]
```diff
@@ -1,17 +1,21 @@
 def test_quiz1_1(page):
-    page.goto("/quiz/Quiz1.html")
+    page.goto("quiz/Quiz1.html")
 
-    page.click("pre[role=\"presentation\"]:has-text(\"​\")")
+    page.click("text=def suma(n,m):")
+    page.press("text=def suma(n,m):", "ArrowDown")
```
Comment (eamanu) [inline tests/test_quiz1.py:5]:
```suggestion
    page.press("text=def suma(n, m):", "ArrowDown")

```

File: tests/test_quiz1.py [hunk 3]
```diff
@@ -1,17 +1,21 @@
 def test_quiz1_1(page):
-    page.goto("/quiz/Quiz1.html")
+    page.goto("quiz/Quiz1.html")
 
-    page.click("pre[role=\"presentation\"]:has-text(\"​\")")
+    page.click("text=def suma(n,m):")
+    page.press("text=def suma(n,m):", "ArrowDown")
+    page.press("text=def suma(n,m):", "Tab")
```
Comment (eamanu) [inline tests/test_quiz1.py:6]:
```suggestion
    page.press("text=def suma(n, m):", "Tab")

```

File: tests/test_quiz1.py [hunk 4]
```diff
@@ -1,17 +1,21 @@
 def test_quiz1_1(page):
-    page.goto("/quiz/Quiz1.html")
+    page.goto("quiz/Quiz1.html")
 
-    page.click("pre[role=\"presentation\"]:has-text(\"​\")")
+    page.click("text=def suma(n,m):")
+    page.press("text=def suma(n,m):", "ArrowDown")
+    page.press("text=def suma(n,m):", "Tab")
+    page.type("text=def suma(n,m):", "return n+m")
```
Comment (eamanu) [inline tests/test_quiz1.py:7]:
```suggestion
    page.type("text=def suma(n, m):", "return n+m")

```

**Review by eamanu — COMMENTED** [review]
Comentarios menores

**Review by eamanu — APPROVED** [review]

File: tests/test_quiz3.py [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    page.type("text=def calcular_cambio(cobro, pago):",
+              "cambio%=2")
+    page.press("text=def calcular_cambio(cobro, pago):",
+               "Enter")
+    page.type("text=def calcular_cambio(cobro, pago):",
+              "billetes.append(cambio//1)")
+    page.press("text=def calcular_cambio(cobro, pago):",
+               "Enter")
+    page.type("text=def calcular_cambio(cobro, pago):",
+              "return billetes")
```
[the remark this answers is not in the document]
Comment (reingart) [reply tests/test_quiz3.py:64]: - [x] Issue LeoCumpli21/PyZombis#27

**Review by reingart — APPROVED** [review]
LGTM

**Pull request merged**